pennylane/interfaces/jax_jit.py

```diff
@@ -160,6 +159,17 @@ def wrapped_exec_fwd(params):
     def wrapped_exec_bwd(params, g):
 
         if isinstance(gradient_fn, qml.gradients.gradient_transform):
+            for t in tapes:
+                multi_probs = (
+                    any(o.return_type is qml.measurements.Probability for o in t.observables)
+                    and len(t.observables) > 1
+                )

```

Comment:
It seems like if there are multiple tapes then this loop will keep overwriting the value of `multi_probs`.

Reply:
Great catch! Thanks.